ofrak_core/ofrak/ofrak_context.py

```diff
@@ -83,12 +83,20 @@ async def shutdown_context(self):
 
 
 class OFRAK:
-    def __init__(self, logging_level: int = logging.WARNING):
+    def __init__(self, logging_level: int = logging.WARNING, audit_dependencies: bool = False):
```

Comment:
What about `skip_incomplete_components` instead of `audit_dependencies`? I find it hard to understand what this flag means without reading the docstring.

Reply:
It's not obvious was "incomplete" means - the most clear flag I think is `skip_components_missing_dependencies`, but I thought that was too long.

Reply:
Gotcha. If we can get the word `skip` into it I think it would make it clearer. Not critical though

Reply:
Sure!